modules/compiler/spirv-ll/source/builder_core.cpp

```diff
@@ -104,9 +104,8 @@ llvm::Error Builder::create<OpSource>(const OpSource *op) {
   source += ", Version: " + std::to_string(op->Version());
 
   if (op->wordCount() > 3) {
-    std::string file_path = module.getDebugString(op->File());
-    if (!file_path.empty()) {
-      source += ", Source file: " + file_path + "\r\n";
+    if (auto file_path = module.getDebugString(op->File())) {
```

Comment:
This is a change in behaviour if the file was provided but was empty. We'd now produce `Source file: `. I couldn't work out if this was better than the current behaviour or not.